iroh/src/services.rs

```diff
@@ -82,7 +82,7 @@ async fn start_services(api: &Api, services: HashSet<&str>) -> Result<()> {
         });
 
     let missing_services: HashSet<&str> = services
-        .difference(missing_services)
+        .intersection(missing_services)
```

Comment:
this was breaking a front-of-house command and had no test. Has a test now, and a fix

Reply:
oops